src/view/dataElements/identityMap.jsx

```diff
@@ -43,9 +44,29 @@ const getInitialValues = ({ initInfo }) => {
     ? identitiesMapToArray(initInfo.settings)
     : [getDefaultIdentity()];
 
-  return {
-    identities
-  };
+  return fetchNamespaces({
+    orgId: initInfo.company.orgId,
+    imsAccess: initInfo.tokens.imsAccess
+  }).then(response => {
+    if (response.length > 0) {
+      const namespaces = response.map(namespace => {
```

Comment:
It's cleaner to remove the return statement:
const namespaces = response.map(namespace => ({
  value: namespace.code,
  label: namespace.code
}));

Reply:
yes, thank you